src/components/RenderRocket.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const RenderRocket = (
+  {
+    name, desc, id, img,
+  },
+) => (
+  <li className="rocket-li">
+    <div className="img-container">
+      <img alt={name} src={img} />
+    </div>
+    <div className="rocket-info">
+      <h2>{name}</h2>
+      <p>{desc}</p>
+      <button type="button" id={id}>
+        Reserve rocket
+      </button>
+    </div>
+  </li>
+);
+
+RenderRocket.propTypes = {
+  name: PropTypes.string.isRequired,
+  desc: PropTypes.string.isRequired,
+  img: PropTypes.string.isRequired,
+  id: PropTypes.string.isRequired,
```

Comment:
There's a console error that states the value of ```id``` is undefined. Maybe you missed passing it as a parameter when you created the component. 

![image](https://user-images.githubusercontent.com/74314917/156246598-92a5046d-74fb-45ba-b8e4-aae227e4453f.png)